src/osd/OSD.cc

```diff
@@ -7284,36 +7284,38 @@ void OSDService::maybe_share_map(
 
   // assume the peer has the newer of the op's sent_epoch and what
   // we think we sent them.
-  session->projected_epoch_lock.lock();
-  if (peer_epoch_lb > session->projected_epoch) {
-    dout(10) << fmt::format(
-       "{}: con {} updating session's projected_epoch"
-       " from {} to ping map epoch of {}",
-       __func__, con->get_peer_addr(),
-       session->projected_epoch, peer_epoch_lb)
-       << dendl;
-    session->projected_epoch = peer_epoch_lb;
-  }
+  epoch_t send_from;
```

Comment:
Initialize this to something.  If there's a constand for an invalid epoch use that.  Otherwise, 0.